daemon/cmd/daemon.go

```diff
@@ -989,7 +984,7 @@ func changedOption(key string, value option.OptionSetting, data interface{}) {
 	d := data.(*Daemon)
 	if key == option.Debug {
 		// Set the debug toggle (this can be a no-op)
-		if d.DebugEnabled() {
+		if option.Config.Opts.IsEnabled(option.Debug) {
 			logging.SetLogLevelToDebug()
```

Comment:
it looks like it's no longer possible to reset the log level to something else than `debug` after this has been changed once. Seems like this was introduced with https://github.com/cilium/cilium/pull/16021 some time ago.